src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -297,18 +318,39 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
         </template>
     </ModalCard>
 
+    <ModalCard v-else-if="activeStep === 'REFRESH_MOD_LIST'" key="REFRESH_MOD_LIST" :is-active="isOpen" :can-close="false">
+        <template v-slot:header>
+            <h2 class="modal-title">Refreshing mod list</h2>
+        </template>
+        <template v-slot:footer>
+            <div>
+                <p>
+                    At least some of the packages were not found. We're trying to fix this by refreshing the online mod list:</br>
+                    {{$store.state.tsMods.thunderstoreModListUpdateStatus}}
+                </p>
+            </div>
+        </template>
+    </ModalCard>
+
     <ModalCard v-else-if="activeStep === 'REVIEW_IMPORT'" key="REVIEW_IMPORT" :is-active="isOpen" @close-modal="closeModal">
         <template v-slot:header>
             <h2 class="modal-title">Packages to be installed</h2>
         </template>
         <template v-slot:body>
-            <OnlineModList :paged-mod-list="knownProfileMods" :read-only="true" />
+            <OnlineModList
+                v-if="profileMods.known.length > 0"
+                :paged-mod-list="knownProfileMods"
+                :read-only="true"
+            />
+            <p v-else>
+                None of the packages in the profile were found on Thunderstore.
+            </p>
             <div v-if="profileMods.known.length === 0 || profileMods.unknown.length > 0" class="notification is-warning margin-top">
                 <p v-if="profileMods.known.length === 0">
-                    None of the packages in the profile were found on Thunderstore:
+                    These packages were not found:
                 </p>
                 <p v-else>
-                    Some of the packages in the profile were not found on Thunderstore:
+                    These packages in the profile were found on Thunderstore:
                 </p>
```

Comment:
I would perhaps reconsider how these three paragraphs work together. First saying no packages were found and then listing all the packages seems redundant. Saying "These packages in the profile were found on Thunderstore" and then listing `unknownProfileModNames` seems like a bug?

Should this perhaps just say  "These packages in the profile were not found on Thunderstore" (notice the "not") inside the yellow box and then list the unknown mods for all cases?